gradio/components/audio.py

```diff
@@ -343,11 +343,20 @@ def postprocess(
         if isinstance(y, tuple):
             sample_rate, data = y
             file_path = self.audio_to_temp_file(
-                data, sample_rate, dir=self.DEFAULT_TEMP_DIR, format=self.format
+                data,
+                sample_rate,
+                format="mp3" if self.streaming else self.format,
             )
             self.temp_files.add(file_path)
         else:
-            file_path = self.make_temp_copy_if_needed(y)
+            if isinstance(y, Path):
+                y = str(y)
+            if self.streaming and not y.endswith(".mp3"):
+                sample_rate, data = processing_utils.audio_from_file(y)
```

Comment:
What's the issue with wav? Do you have an example where it fails? cantina.wav works fine for me without this.

Reply:
really? cantina.wav specifically does not work for me on chrome. It may be differ browser to browser - the problem with streaming wav's is that wav files have a header in each chunk that can confuse the browser when they are naively chained to gether as a stream - mp3's do not have this problem. As explained to me by chatgpt here: https://chat.openai.com/share/dc4b8dd7-faf3-43f6-98df-ea8d7da23fcc